.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,85 @@
+name: Bug Report
+description: File a bug report to help us improve
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: "# Please search the issue tracker for existing bug reports before submitting your own."
+  - type: input
+    id: system-information
+    attributes:
+      label: System Information
+      description: |
+        - The operating system you use to run LMMS.
+        - When relevant, also include your hardware information.
+      placeholder: ex. Fedora Linux 39, KDE Plasma 5.27.10 - 13th Gen Intel® Core™ i9-13950HX, 32GB RAM
+    validations:
+      required: true
+  - type: input
+    id: affected-version
+    attributes:
+      label: LMMS Version(s)
+      description: |
+        - The version of LMMS affected by the bug.
+        - Can be an official version number, nightly release identifier, or commit hash.
+        - The version number can be found under the Help > About menu.
+      placeholder: ex. 1.2.2, 1.3.0-alpha.1.518+gdd53bec31, 2d185df
+    validations:
+      required: true
+  - type: input
+    id: working-version
+    attributes:
+      label: Most Recent Working Version
+      description: |
+        - If there is a previous version of LMMS that did not exhibit the bug, please include it here.
+        - Using `git bisect` to determine exactly where the regression occurred will make it easier for LMMS developers to find and fix the bug.
```

Comment:
I worry mentioning `git bisect` could confuse or scare off less technical users. Those who know how to use it probably know to do so anyway. Also, it reports the first bad commit, not the last good one, so we risk people putting the wrong version in this field.